scripts/evaluate_model.py

```
@@ -0,0 +1,530 @@
+        config={
+            "model_name": model_name,
+            "batch_size": batch_size,
+            "chunk_len_sec": chunk_len_sec,
+            "word_timestamps": word_timestamps,
+        },
+        audio_path=str(audio_path),
+        task="evaluate_model",
+    )
+
+    try:
+        wav, sample_rate, segments, load_elapsed, duration_sec = _load_and_prepare_audio(
```

Ruff (0.15.20)

[warning] Unpacked variable `wav` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)